templates/default/rt/rtprintindex.html

```diff
@@ -134,35 +134,23 @@ <H1>{$layout.pagetitle}</H1>
 
 				</TR>
 				{/if}
-				<TR>
-					<TD width="1%" COLSPAN="2" NOWRAP>
-						<IMG src="img/queue.gif" alt=""><B>{trans("Categories:")}</B>
-					</TD>
-					<TD width="1%">
-						<TABLE>
-							<TBODY class="lms-ui-multi-check">
-							{foreach $categories as $category}
-							<TR>
-								<TD width="1%" align="center">
-									<INPUT type="checkbox" name="categories[{$category.id}]" class="lms-ui-multi-check" value="1" CHECKED>
-								</TD>
-								<TD width="1%">
-									<label for="cat1{$category.id}">{$category.name}</label>
-								</TD>
-							</TR>
+				<tr>
+					<td width="1%">
+						{icon name="helpdesk"}
+					</td>
+					<td class="bold nobr">
+					    {trans("Category")}
+					</td>
+					<td>
+						<input type="hidden" name="catids" value="all">
+						<select size="1" name="catids[]" multiple class="lms-ui-multiselect lms-ui-multiselect-filter" data-default-value="{trans("- all -")}">
+							<option value="-1">{trans('- no categories -')}</option>
+							{foreach $categories $c}
```

Comment:
Czy rzeczywiście smarty pozwala na taką sładnię przy foreach - nawet bez `as`?
Jest to oficjalnie udokumentowane?

Reply:
Tak, mamy w filtrach obecnie to jest copy&paste.

Reply:
Wrzuciłem "as"